tests/manage/mcg/helpers.py

```diff
@@ -154,6 +163,152 @@ def upload_parts(mcg_obj, awscli_pod, bucketname, object_key, body_path, upload_
     return parts
 
 
+def oc_create_aws_backingstore(cld_mgr, backingstore_name, uls_name, region):
+    """
+    Create a new backingstore with aws underlying storage using oc create command
+
+    Args:
+        cld_mgr (CloudManager): holds secret for backingstore creation
+        backingstore_name (str): backingstore name
+        uls_name (str): underlying storage name
+        region (str): which region to create backingstore (should be the same as uls)
+
+    """
+    bs_data = templating.load_yaml(constants.MCG_BACKINGSTORE_YAML)
+    bs_data['metadata']['name'] = backingstore_name
+    bs_data['metadata']['namespace'] = config.ENV_DATA['cluster_namespace']
+    bs_data['spec']['awsS3']['secret']['name'] = cld_mgr.aws_client.get_oc_secret()
```

Comment:
Access the secret name directly - 
`bs_data['spec']['awsS3']['secret']['name'] = cld_mgr.aws_client.secret.name`